extensions/amp-story/1.0/amp-story-grid-layer.js

```diff
@@ -28,7 +28,12 @@
 
 import {AmpStoryBaseLayer} from './amp-story-base-layer';
 import {assertDoesNotContainDisplay, setStyles} from '../../../src/style';
-import {matches, scopedQuerySelectorAll} from '../../../src/dom';
+import {
```

Comment:
nit: this all looks pretty good to me, but I'm wondering if we should make it available more widely than just `amp-story-grid-layer`!

Should we add this in CTA layers as well?  Page attachments?

/cc @estherkim @gmajoulet

Reply:
Great idea and I don't see why not. I moved the styling method to `utils.js` so that descendant elements of all 3 (`amp-story-grid-layer`, `amp-story-cta-layer`, and `amp-story-page-attachment`) can use it.